table/server/master/src/main/java/alluxio/master/table/Table.java

```diff
@@ -96,10 +98,17 @@ private Table(Database database, UdbTable udbTable, @Nullable Table previousTabl
         }
         partitions.add(newPartition);
       }
+      LOG.debug("Get {} partitions from Table {} Database {}.",
```

Comment:
maybe "Updating existing table {}.{} with {} total partitions." ? This could just be at INFO level, since it won't be too noisy.

Reply:
Changed.